src/app/pages/Houses/useHousePoints.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+
+const FEED_URL = "https://docs.google.com/spreadsheets/d/";
+const SPREADSHEET_KEY = "1AV98P5Fs3INBtbwT69oJvfN0eAgj9daVw6o_6XIGEiI";
+const MODE = "pub";
+const SHEET_GID = "1705286477";
+
+function useHousePoints() {
+	const [housePoints, setHousePoints] = useState({});
```

Comment:
For my understanding, what is the benefit of this?

Reply:
just good practice to not declare new objects,arrays,functions inside a react function. You'll get miniscule, if any, benefit here but if this component ever rerenders, it'll be recreating the empty object for no good reason

Reply:
if this were a more non-static object, you'd wrap it in a useMemo. if it were a function, you'd wrap it in a useCallback. since it's a static object, you can declare it outside of this function and use it inside. That way, when this component rerenders, it'll never have to recreate the object.

Reply:
Got it, thanks for the clarification.

Reply:
Extracted initial value in ffa5f5b.